pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -168,6 +172,12 @@ type PrometheusConfig struct {
 	EnableRemoteWriteReceiver bool `json:"enableRemoteWriteReceiver,omitempty"`
 }
 
+type AlertmanagerConfig struct {
+	// Disables the deployment of Alertmanager.
+	// +optional
+	Disabled bool `json:"disabled,omitempty"`
```

Comment:
can we also set `default: false`?

Reply:
Is that required? It's the zero value.

Reply:
> Is that required? It's the zero value.

agreed, I don't feel we need to specify a default.

Reply:
We need to specify default so that it appears when `oc get -o yaml` an already applied resource where user has relied on the defaults. see: replicas e.g.

Reply:
OK